biz.aQute.bnd.exporters/test/biz/aQute/bnd/exporters/subsystem/SubsystemTest.java

```diff
@@ -0,0 +1,170 @@
+package biz.aQute.bnd.exporters.subsystem;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+import org.osgi.service.subsystem.SubsystemConstants;
+
+import aQute.bnd.build.Project;
+import aQute.bnd.build.Run;
+import aQute.bnd.build.Workspace;
+import aQute.bnd.exporter.subsystem.SubsystemExporter;
+import aQute.bnd.osgi.Jar;
+import aQute.bnd.osgi.JarResource;
+import aQute.bnd.osgi.Resource;
+import aQute.lib.io.IO;
+import junit.framework.TestCase;
+
+public class SubsystemTest extends TestCase {
```

Comment:
I think you should either use Junit4 or Junit3. Not both :-) So remove the extends TestCase and add @Before to setUp.